### PR TITLE
Sign In Screenshots are outdated

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation task: https://mattermost.atlassian.net/browse/MM-33036

Updated:
- User Guide > Getting Started > Signing In
   - Updated screenshots for the following subsections:
      - Email Address or Username Sign In
      - AD/LDAP Sign In (SSO)
      - SAML Single Sign-On (SSO)
      - Switching Teams

### CODE REVIEW

**Comment by cwarnermm** [comment]
@esethna - Do you happen to have updated screenshots for GitLab, Google, and Office 365 sign in screens available?

File: source/help/getting-started/signing-in.rst [hunk 1]
```diff
@@ -28,20 +28,18 @@ Email Address or Username Sign In
 
 When enabled by your System Admin, you can sign in with the username or email address used to create your account.
 
-If you have forgotten your password, you can reset it by clicking **I forgot my password** on the sign-in screen, or contact your System Admin for help resetting your password.
+If you have forgotten your password, you can reset it by selecting **I forgot my password** on the sign-in screen, or contact your System Admin for help resetting your password.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:31]:
```suggestion
If you've forgotten your password, you can reset it by selecting **I forgot my password** on the sign-in screen, or contact your System Admin for help resetting your password.
```

File: source/help/getting-started/signing-in.rst [hunk 2]
```diff
@@ -28,20 +28,18 @@ Email Address or Username Sign In
...
 
 .. image:: ../../images/sign-in_with_email.png
-  :width: 239px
-  :height: 232px
+  :alt: Sign in to Mattermost with your username or email address, or reset your password.
 
 GitLab Single Sign-On (SSO)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When enabled by your System Admin, you can sign in using your GitLab account using a one-click sign in option.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:39]:
```suggestion
When enabled by your System Admin, you can sign in with your GitLab account using a one-click sign in option.
```

File: source/help/getting-started/signing-in.rst [hunk 3]
```diff
@@ -50,9 +48,8 @@ Google Single Sign-On (SSO)
 
 When enabled by your System Admin, you can sign in using your Google account using a one-click sign in option.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:49]:
```suggestion
When enabled by your System Admin, you can sign in with your Google account using a one-click sign in option.
```

File: source/help/getting-started/signing-in.rst [hunk 4]
```diff
@@ -61,9 +58,8 @@ Office 365 Single Sign-On (SSO)
 
 When enabled by your System Admin, you can sign in using your Office 365 account using a one-click sign in option.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:59]:
```suggestion
When enabled by your System Admin, you can sign in with your Office 365 account using a one-click sign in option.
```

**Review by justinegeffen — COMMENTED** [review]
Some minor suggestions and I do have one question:
We refer to the "sign-in" page, but other references to "sign in" are not hyphenated. I noticed that we refer to the to "sign-in" page in other docs so I am happy keeping it as such. But should we carry that over to the non-hyphenated instances?

**Comment by cwarnermm** [comment]
Great question, @justinegeffen. I've also noticed this inconsistency, and I'm happy to align on a standard. 
@esethna Do you have a preference, for end users, on whether we refer to our Mattermost sign in page as a "sign-in" or "sign in" page?

**Comment by justinegeffen** [comment]
To expand on my comment, I propose leaving "sign-in" as-is, when referring to the "sign-in page" and "sign in" in other instances. That's the current standard, so as long as we all agree on that, it'll be one less chunk of work we have to do.

**Comment by esethna** [comment]
@cwarnermm works for me :)

**Review by esethna — APPROVED** [review]

**Comment by cwarnermm** [comment]
Thanks, @esethna! I've logged https://mattermost.atlassian.net/browse/MM-33322 to update the User's Guide documentation to be more consistent. CC @justinegeffen

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**